packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/edit-display-name/routing.js

```diff
@@ -0,0 +1,39 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'app.dedicated-cluster.cluster.dashboard.edit-display-name',
+    {
+      url: '/edit-display-name',
+      component: 'managerDedicatedClusterDashboardEditDisplayNameComponent',
+      layout: 'modal',
+      params: {
+        displayName: null,
+      },
+      resolve: {
+        displayName: /* @ngInject */ ($transition$) =>
+          $transition$.params().displayName,
+        goBack: /* @ngInject */ (goToClusterGeneralInfo) =>
+          goToClusterGeneralInfo,
+        handleError: /* @ngInject */ (Alerter, goBack) => (error) => {
+          Alerter.error(
+            error.message || error.data?.message,
```

Comment:
sounds more logic to check `error.data` before. And what about if no message ?
```suggestion
            error.data?.message || error.message || error,
```